bootstrap-orbit.sh

```diff
@@ -131,3 +131,23 @@ if [[ $DONT_COMPILE != "yes" ]]; then
   fi
 fi
 
+PRE_COMMIT_HOOK_FILE=.git/hooks/pre-commit
+
+function install_clang_format_pre_commit_hook {
+  if [ ! -f $PRE_COMMIT_HOOK_FILE ] ; then
+    touch $PRE_COMMIT_HOOK_FILE
+    chmod +x $PRE_COMMIT_HOOK_FILE
+  fi
+  echo "source \"contrib/hooks/clang-format-pre-commit.sh\"" >> $PRE_COMMIT_HOOK_FILE
+}
+
+if  [ ! -f $PRE_COMMIT_HOOK_FILE ] || ! grep -q "contrib/hooks/clang-format-pre-commit.sh" .git/hooks/pre-commit ; then
```

Comment:
This is probably missing a check whether it's a git repo in the first place (whether `.git` exists).

Reply:
Done.